tensorrt_llm/_torch/pyexecutor/py_executor.py

```diff
@@ -457,7 +457,11 @@ def _process_iter_stats(self, finished_requests, scheduled_batch,
             self._update_iter_stats(iter_stats, iter_latency_ms,
                                     len(finished_requests), scheduled_batch))
 
-        return
+    def _common_executor_loop_cleanup(self):
```

Comment:
nit: maybe no need for "common"? just _executor_loop_cleanup() sounds good

Reply:
Makes sense, renamed.